pyproject.toml

```diff
@@ -0,0 +1,94 @@
+[build-system]
+requires = ["setuptools>=67", "setuptools_scm[toml]>=6.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "maraboupy"
+version = "1.0.0"
+authors = [
```

Comment:
Hi Wen, thanks for the updates. My final request is that could the authors just be "the Marabou development team", optionally with a link to https://github.com/NeuralNetworkVerification/Marabou/blob/master/AUTHORS ?

This is what Clark suggested.

Reply:
It won't be possible to link to the authors file, but I can at least package it.